packages/components/src/utils/hooks/use-jump-step.js

```diff
@@ -29,7 +29,10 @@ function useJumpStep( {
 	const [ isShiftKey, setIsShiftKey ] = useState( false );
 
 	useEffect( () => {
-		const handleShiftKeyToggle = ( event ) => {
+		const handleShiftKeyToggle = (
+			/** @type {KeyboardEvent} */
```

Comment:
This get's passed to `window.addEventListener` so we do _not_ use the synthetic version from `react`